topics/ui/controls/checkbox.topic

```diff
@@ -0,0 +1,207 @@
+                        </td>
+                        <td width="378">
+                                <format color="E55765" style="bold">Incorrect</format><img src="checkbox_label_right_incorrect.png"
+                                                                                           alt="An incorrect checkbox with the label on the left"/>
+                        </td>
+                    </tr>
+                </table>
+            <tabs group="languages">
+                <tab title="Kotlin UI DSL" group-key="kotlin">
+                    <code-block lang="kotlin">
+                        checkBox(
+                        &quot;&quot;&quot;&lt;html&gt;Insert selected suggestion by pressing space, dot,&lt;br/&gt;
+                        or other context-dependent keys&lt;/html&gt;&quot;&quot;&quot;)
+                    </code-block>
+                </tab>
+                <tab title="Java" group-key="java">
+                    <code-block lang="java">
+                        new JCheckBox(
+                        &quot;&lt;html&gt;Insert selected suggestion by pressing space, dot,&lt;br/&gt;&quot; +
+                        &quot;or other context-dependent keys&lt;/html&gt;&quot;);
+                    </code-block>
+                </tab>
+            </tabs>
+        </chapter>
+        <chapter title="Long labels">
+            <p>If a label is long, split it into two lines. Use HTML formatting for that. Avoid labels that take more than two lines. See recommendations on <a anchor="writing-guidelines">writing concise labels</a> below.</p>
+                <table style="none" border="false">
+                    <tr>
+                        <td width="378">
+                            <format color="369650" style="bold">Correct</format><img src="checkbox_label_long_correct.png"
+                                                                                     alt="A correct checkbox with the label on the right"/>
+                        </td>
+                        <td width="378">
+                                <format color="E55765" style="bold">Incorrect</format><img src="checkbox_label_long_incorrect.png"
+                                                                                           alt="An incorrect checkbox with the label on the left"/>
+                        </td>
+                    </tr>
+                </table>
+            </chapter>
+        <chapter title="Checkboxes in a table">
+            <p>If a checkbox appears in a table, place the label into the column header and do not repeat it on every
+                row:</p>
+            <img src="checkbox_when_to_use_3.png"
+                 alt="A table with checkboxes where the label is placed into the column header" width="706"/>
+                <chapter title="Implementation" collapsible="true" id="checkbox-table-implementation">
```

Comment:
Alignment of the "Implementation" section seems off, added screenshot to the Figma file under the checkboxes pics

Reply:
Yeah, I think we should discuss it separately if the placement of the "+" buttons in collapsible sections can be changed